framework/sac_rlhf.py

```diff
@@ -8,12 +8,15 @@
 
 import gymnasium as gym
 import numpy as np
+import shimmy
```

Comment:
```suggestion
import shimmy # noqa
```

`#noqa` is probably the thing we need here (it means "no quality assurance", the naming is a bit weird here)